client/src/app/ui/modules/search-selector/components/base-search-selector/base-search-selector.component.html

```diff
@@ -68,6 +68,7 @@
         >
             {{ noOptionsFoundLabel | translate }}
         </mat-option>
+        <mat-option *ngIf="addClearSelection" translate>Clear selection</mat-option>
```

Comment:
for consistency-sake I would suggest a single minus 

```suggestion
        <mat-option *ngIf="addClearSelection"> - </mat-option>
```

Reply:
done